src/app/parcels/edit/editParcelForm.tsx

```diff
@@ -41,24 +43,28 @@ const EditParcelForm = ({ parcelId }: EditParcelFormProps): React.ReactElement =
     const [collectionCentres, setCollectionCentres] = useState<CollectionCentresLabelsAndValues>(
         []
     );
+    const [packingSlots, setPackingSlots] = useState<PackingSlotsLabelsAndValues>([]);
+    const [packingSlotShown, setPackingSlotsShown] = useState<boolean | undefined>(true);
```

Comment:
What does this state really mean? Does it signify if the current packing slot of the parcel has `is_shown` set to true or not?